lib/brain.js

```diff
@@ -1951,6 +1951,9 @@ Brain.prototype._billRead = function _billRead (data) {
   if (!this.isCompliant(amount)) {
     // Cancel current bill
     this.billValidator.reject()
+
+    this._setState('smsVerification')
+
```

Comment:
Should use ``_transitionState`` instead of ``_setState`` and ``browser().send``.